content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -130,31 +92,28 @@ can set the number with `--jobs` (`-j`).
 If an experiment being pulled already exists in the local project, DVC won't
 overwrite it unless you supply `--force`.
 
-### Example: Pushing or pulling multiple experiments
+## 👨‍🏫 Example: Sharing multiple experiments
```

Comment:
let's do emojis as a separate PR?

do we have links to these examples in the text? 

should we after all embed examples into the text (if you read the book do you expect examples at the end?)

Reply:
Hmmm. OK removing emojis for now but we need an issue then to address this in all guides. I wonder if it's really that important

There are no manual links to these examples anywhere but the ToC nav shows them.

And yes, there are short examples or what I call "code samples" embedded in the text as needed.

These examples by the end are interesting applications that don't map exactly to specific paragraphs from the previous sections. In fact the last example is the longest section in the whole doc.